src/pages/explore/ui/stats.tsx

```diff
@@ -0,0 +1,81 @@
+import { round } from '@penumbra-zone/types/round';
+import { Text } from '@penumbra-zone/ui/Text';
+import { InfoCard } from './info-card';
+import { pluralizeAndShortify } from '@/shared/utils/pluralize';
+import { shortify } from '@penumbra-zone/types/shortify';
+import { getFormattedAmtFromValueView } from '@penumbra-zone/types/value-view';
+import { getStats } from '../api/get-stats';
+
+export const ExploreStats = async () => {
+  const stats = await getStats();
+
+  if ('error' in stats) {
+    return (
+      <Text large color='destructive.light'>
+        {stats.error}
+      </Text>
+    );
+  }
+
+  return (
+    <div className='grid grid-cols-1 desktop:grid-cols-3 gap-2'>
+      <InfoCard title='Total Trading Volume (24h)'>
+        <Text large color='text.primary'>
+          <Text large color='text.primary'>
+            {shortify(Number(getFormattedAmtFromValueView(stats.directVolume)))}
```

Comment:
@grod220 the volume comes a 10^21 number, and even the `ValueViewComponent` displays it as `1589T`. Something seems to be wrong in these calculations. Do you know what?

Reply:
Would also highlight that the `liquidity` field returns something like `liquidity: 118364618673.38`, which is not an integer. It requires us to `parseInt()`, which may fail with big numbers

Reply:
For mainnet data, I'm only seeing weird values for the month window in the aggregate summary, will investigate, but probably related to needing some more stringent filtering of assets with no real quote asset price on penumbra.

The liquidity field could be amended to not be a float, but every other value that is conceptually an Amount (like volume) is already a float, and so I think I would recommend just developing a bit of infrastructure here to deal with that. We need bespoke code here to deal with prices, which have to be a float, as well.